src/module.c

```diff
@@ -384,7 +384,7 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
             RedisModule_ReplyWithSimpleString(ctx, "size");
             RedisModule_ReplyWithLongLong(ctx, chunkSize);
             RedisModule_ReplyWithSimpleString(ctx, "bytesPerSample");
-            RedisModule_ReplyWithDouble(ctx, chunkSize / numOfSamples);
+            RedisModule_ReplyWithDouble(ctx, (float)chunkSize / numOfSamples);
```

Comment:
@danni-m I guess this commit is not related (and removes a warning), does it fix a bug?

Reply:
@gkorland its not totally related, but I found this while making sure the docs align with the actual output.

Reply:
How come it didn't fail any test? should we change any test to cover this change?